packages/fxa-auth-server/test/local/routes/subscriptions/stripe-webhooks.js

```diff
@@ -486,6 +517,78 @@ describe('StripeWebhookHandler', () => {
       });
     });
 
+    describe('handleProductUpdatedEvent', () => {
+      let scopeContextSpy, scopeSpy;
+      beforeEach(() => {
+        scopeContextSpy = sinon.fake();
+        scopeSpy = {
+          setContext: scopeContextSpy,
+        };
+        sandbox.replace(Sentry, 'withScope', (fn) => fn(scopeSpy));
+      });
+
+      it('throws a sentry error if the update event data is invalid', async () => {
+        const updatedEvent = deepCopy(eventProductUpdated);
```

Comment:
Don't know about other devs, but I usually expect mock data to be valid by default.  Either way though, since it is not valid, maybe update the name to say so?

Reply:
I will make the mock event valid and set it to invalid when necessary in tests